### DISCUSSION TITLE
DateTime field can store the UTC time into the database?

### DISCUSSION THREAD

**Discussion by harshalizode** [dymmond/mongoz#34] [Q&A]
Hello,
I have a question about the DateTime field. When I store datetime data, it is stored in ISO format; however, I want to store it in UTC format; are there any settings to modify the format while storing datetime data?
Thnak you.

**Comment by tarsil** [comment]
This is actually using the motor behind the scenes but internally we don't have any "format" field being sent and validated. 

This is actually a nice feature to add. 

Could you provide me some examples of the formats you would like to save? This can be used to implement this feature however possible

**Reply by harshalizode** [reply]
Yeah, We can store the UTC format also, because UTC seems to be a very stable candidate with good timezone translation support and UTC it will be easier format to export and import as UTC is the most reliable time zone.
Thank you.

**Reply by tarsil** [reply]
Is it working already? Of not. Please provide me some examples

**Comment by harshalizode** [comment]
Yes, it stores the expected format, however it gets confused with the ISODate data type when it retrieves the datetime object from MongoDB. 
Thank you for your response.
